src/types/CaseListQueryParams.ts

```diff
@@ -1,6 +1,11 @@
 export type QueryOrder = "asc" | "desc"
 
-export type Reason = "Triggers" | "Exceptions" | "Bails"
+export type Reason = Reasons
+export enum Reasons {
```

Comment:
I wonder if we could avoid having both Reason and Reasons defined, I get that one is a type the other is an enum, but it was quite unclear from the naming until I looked it up

Reply:
Yeah I don't like this either.

Reply:
Nice one! 🎉